build.gradle

```diff
@@ -10,9 +10,11 @@ ext {
       compiletesting: 'com.google.testing.compile:compile-testing:0.10',
       javaPoet: 'com.squareup:javapoet:1.8.0',
       moshi: 'com.squareup.moshi:moshi:1.3.1',
+      rxjava: 'io.reactivex:rxjava:1.2.3',
```

Comment:
So we force user to use RxJava? I guess @martijnwalraven had some concerns about this.
I see cases where regular callbacks can be easy for user without Rx knowledge, but not sure